packages/core/test/actions.test.ts

```diff
@@ -2314,6 +2315,72 @@ describe('sendTo', () => {
 
     interpret(parentMachine).start();
   });
+
+  it('should be able to send an event to itself', (done) => {
+    const machine = createMachine({
+      initial: 'a',
+      states: {
+        a: {
+          entry: sendTo(
+            SpecialTargets.Internal,
```

Comment:
This pattern should get into the docs somewhere.

Reply:
This is only for tests; as discussed, we want to encourage raising over sending.

Reply:
IIRC we've discussed that `raise` should be able to raise delayed events. Is this already supported? I think I didn't see any tests like that.

Reply:
Yes this is done